lua/smart-splits/mux/tmux.lua

```diff
@@ -149,7 +149,7 @@ function M.split_pane(direction, size)
 end
 
 function M.on_init()
-  local pane_id = M.current_pane_id()
+  local pane_id = os.getenv('TMUX_PANE')
```

Comment:
Environment variables are available via `vim.env`

```suggestion
  local pane_id = vim.env.TMUX_PANE
```

Also do you think this should be used everywhere, or is this just a special case? I'm not a tmux user so I'm not too sure. If you think this will work everywhere instead of `current_pane_id()`, we can remove that function entirely.

Reply:
I actually tried this. But it seems you coded some logic that verifies whether a move has been successful by comparing the active pane to the supposed target. For that we actually want the active pane. So init and exit are the only places where this plays a role.

Reply:
Ah okay gotcha, that makes sense. This PR looks good then 👍